app/views/research/experiment_solutions/_solution.html.haml

```diff
@@ -1,6 +1,5 @@
 - solution.latest_files.each do |filename, code|
-  .code-editor.js-code-editor{data: { language: solution.language_slug, filename: filename } }
-    = code
+  .code-editor.js-code-editor{data: { language: solution.language_slug, filename: filename } }= code
```

Comment:
@kntsoriano is it worth adding a test for this. It would be a really easy thing to regress. Maybe make a note that when everything is stable, this is something we write a test to check?